alvr/packets/src/lib.rs

```diff
@@ -188,6 +188,7 @@ pub enum ClientListAction {
     RemoveEntry,
     UpdateCurrentIp(Option<IpAddr>),
     SetConnectionState(ConnectionState),
+    ToggleCabled,
```

Comment:
I believe this variant should take a bool. It's because the behavior might get confusing if there are two dashboards concurrently calling this.